server/routerlicious/packages/routerlicious-base/src/riddler/api.ts

```diff
@@ -47,15 +47,17 @@ export function create(
      */
     router.get("/tenants/:id", (request, response) => {
         const tenantId = getParam(request.params, "id");
-        const tenantP = manager.getTenant(tenantId);
+        const includeDisabled = request.get("Include-Deleted")?.toLowerCase() === "true";
```

Comment:
May be we should call the header 'Include-disabled' ? because technically the tenant would be disabled and marked for deletion, not actually deleted.

Reply:
Thanks. I updated this in the latest iteration.